src/Components/ToDoContainer/ToDoApp.js

```diff
@@ -0,0 +1,116 @@
+import './ToDoApp.css';
+import React, { Component } from 'react';
+import { v4 as uuidv4 } from 'uuid';
+import ToDoList from '../ToDoList/ToDoList';
+import Header from '../Header/Header';
+
+export default class ToDoContainer extends Component {
```

Comment:
It will be best if you use function-based component instead of class-based components

Reply:
Thank you for the advice @Aseadd!